pyproject.toml

```diff
@@ -53,6 +53,11 @@ dependencies = [
     'igakit @ https://github.com/dalcinl/igakit/archive/refs/heads/master.zip'
 ]
 
+[project.optional-dependencies]
+test = [
+    "pytest-cov >= 5.0.0",
+]
+
```

Comment:
The packages `pytest` and `pytest-xdist` are always installed, because they are listed under `project.dependencies`. Do you want to list them here as well?

Reply:
Fixed